deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -694,3 +536,274 @@ def merge_input_provider(index, item):
         round_completion_file_s3_url,
         is_inplace_compacted,
     )
+
+
+def _process_merge_results(
```

Comment:
reorder the methods in the same order as they are called. This avoid scrolling down reading the code. Even better is if you can move these to separate modules.